project/profiles/capacity4more/themes/c4m/kapablo/templates/page.tpl.php

```diff
@@ -106,6 +106,12 @@
   <?php endif; ?>
 
   <div id="header-ec-wrapper">
+    <?php if (isset($image_banner)): ?>
+      <div class="row background-banner">
```

Comment:
@nirgn975 
Please remove the `row` class since we don't use bootstrap grid over here anyway ...